<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Achitp</class>
 <widget class="QMainWindow" name="Achitp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Achitp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	background: url(':/grid');
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="Hole" name="hole00">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
      </property>
      <property name="icon">
       <iconset resource="Achitp.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Hole" name="hole01">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
      </property>
      <property name="icon">
       <iconset resource="Achitp.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="Hole" name="hole02">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
      </property>
      <property name="icon">
       <iconset resource="Achitp.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Hole" name="hole10">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
      </property>
      <property name="icon">
       <iconset resource="Achitp.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="Hole" name="hole11">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
      </property>
      <property name="icon">
       <iconset resource="Achitp.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="Hole" name="hole12">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
      </property>
      <property name="icon">
       <iconset resource="Achitp.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="Hole" name="hole20">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
      </property>
      <property name="icon">
       <iconset resource="Achitp.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="Hole" name="hole21">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
      </property>
      <property name="icon">
       <iconset resource="Achitp.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="Hole" name="hole22">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	outline: none;
}</string>
      </property>
      <property name="icon">
       <iconset resource="Achitp.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Hole</class>
   <extends>QPushButton</extends>
   <header location="global">Hole.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Achitp.qrc"/>
 </resources>
 <connections/>
</ui>
